### PR TITLE
chore: Mark autogenerated release blog post as draft

### PR DESCRIPTION
**Pull request by nzakas**
#### Prerequisites checklist

- [x] I have read the [contributing guidelines](https://github.com/eslint/eslint/blob/HEAD/CONTRIBUTING.md).

#### What is the purpose of this pull request? (put an "X" next to an item)

[ ] Documentation update
[ ] Bug fix ([template](https://raw.githubusercontent.com/eslint/eslint/HEAD/templates/bug-report.md))
[ ] New rule ([template](https://raw.githubusercontent.com/eslint/eslint/HEAD/templates/rule-proposal.md))
[ ] Changes an existing rule ([template](https://raw.githubusercontent.com/eslint/eslint/HEAD/templates/rule-change-proposal.md))
[ ] Add autofix to a rule
[ ] Add a CLI option
[ ] Add something to the core
[x] Other, please explain:

Updated draft blog post template

#### What changes did you make? (Give an overview)

I updated the autogenerated blog post template to include `draft: true` in the frontmatter. This will prevent the blog post from being published until someone manually removes that line during the release process.

### CODE REVIEW

**Comment by aladdin-add** [comment]
just wondering if we should add an item to the release guide and auto-created release issues, if it needs to be removed manually.

**Review by btmills — APPROVED** [review]
> and auto-created release issues

Good idea. I could definitely see myself forgetting that. I updated the release issue template in https://github.com/eslint/eslint-github-bot/pull/176.

> the release guide

Probably worth doing as well, though I don't think that should block this change. LGTM as-is so it can go in if there's not time. That page could use a refresh anyway since it's been a long time since we used [two-person release teams](https://eslint.org/docs/latest/maintainer-guide/releases#release-team).

**Pull request merged**
